internal/support/notifications/init.go

```diff
@@ -95,9 +94,6 @@ func Init(useRegistry bool) bool {
 		registerUpdates = make(chan interface{})
 		go listenForConfigChanges()
 	}
-
-	go telemetry.StartCpuUsageAverage()
```

Comment:
Why was this removed here?

Reply:
My guess is the IOTech branch hadn't been rebased in a bit and I missed it in the merge. I will correct.